v1/terraform.go

```diff
@@ -0,0 +1,92 @@
+package atlas
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+)
+
+// TerraformConfigVersion represents a single uploaded version of a
+// Terraform configuration.
+type TerraformConfigVersion struct {
+	Version   int
```

Comment:
Yeah I'm assuming rails will just ignore it. I just wanted to use the same structure to write.